web/massastation/src/components/SidePanels/Upload/Upload.tsx

```diff
@@ -185,6 +185,10 @@ export default function Upload() {
   const { data: accounts = [] } = useResource<AccountObject[]>(
     `${URL.WALLET_BASE_API}/${URL.WALLET_ACCOUNTS}`,
   );
+  const nicknameInAccounts = accounts.find(
+    (account) => account.nickname === nickname,
+  );
+  if (!nicknameInAccounts) setNickname(accounts[0]?.nickname || '');
```

Comment:
You can't set an `''` value when the user can't be found. 🤔 

What can we do when `accounts[0]?.nickname` is undefined?

Reply:
technically this does not happen because There is a `create an account` button, ill see how I can make this better

Reply:
> technically this does not happen because There is a `create an account` button, ill see how I can make this better

Sure! But be careful, as this condition could create UI problems in the future.

**EX:**  The user decides to open the left panel, but (for some reason) his name doesn't appear in the list.

Reply:
I added a condition in the if to avoid doing this

Reply:
> I added a condition in the if to avoid doing this

Great job! Thanks for that! 🎉 😉